credentials/alts/internal/authinfo/authinfo_test.go

```diff
@@ -139,5 +146,8 @@ func (s) TestALTSAuthInfo(t *testing.T) {
 		if got, want := authInfo.PeerRPCVersions(), tc.outPeerRPCVersions; !reflect.DeepEqual(got, want) {
 			t.Errorf("authinfo.PeerRpcVersions()=%v, want %v", got, want)
 		}
+		if got, want := authInfo.PeerAttributes(), tc.outPeerAttributes; !reflect.DeepEqual(got, want) {
```

Comment:
Instead of checking against the empty map, you can add some random values to your test map, and see if you can get the same value by calling the getter.

Reply:
Sounds good thank you for the input.